Library/Homebrew/compat/download_strategy.rb

```diff
@@ -204,12 +211,6 @@ def scp_source
 class DownloadStrategyDetector
   class << self
     module Compat
-      def detect(url, using = nil)
-        strategy = super
-        require_aws_sdk if strategy == S3DownloadStrategy
```

Comment:
Was the `require_aws_sdk` method removed?

Reply:
Yeah, it looks like it got taken out during deprecation. I linked the commit in the PR description but here again for reference
https://github.com/Homebrew/brew/commit/599ecc9b5ad7951b8ddc51490ebe93a976d43b29

Reply:
@amancevice My bad, could you re-add that method into this `Compat` module instead?